pkg/operator/endpoints/deploy.go

```diff
@@ -47,10 +47,14 @@ func Deploy(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	projectBytes, err := files.ReadReqFile(r, "project.zip")
-	if err != nil {
-		RespondError(w, errors.WithStack(err))
-		return
+	projectBytes := []byte{}
+
+	if _, ok := r.MultipartForm.File["project.zip"]; ok {
+		projectBytes, err = files.ReadReqFile(r, "project.zip")
```

Comment:
nit: I think `ReadReqFile` returns nil if no file, so we might be able to drop the if. If you think it adds clarity feel free to leave it.